pkg/kubelet/cm/cpumanager/cpu_manager.go

```diff
@@ -239,6 +239,8 @@ func (m *manager) Start(activePods ActivePodsFunc, sourcesReady config.SourcesRe
 		return err
 	}
 
+	klog.V(4).InfoS("CPU manager started successfully", "policy", m.policy.Name())
+
 	m.allocatableCPUs = m.policy.GetAllocatableCPUs(m.state)
```

Comment:
can we move the log line here?
```
klog.V(4).InfoS("CPU manager started", "policy", m.policy.Name(), "allocatableCPUs", m.allocatableCPUs)
```

I'm intentionally dropping "succesfully" because it can safely inferred implicitely ("started" + it's Info [vs Error]) and we save some arguably redundant bytes in the log entry.

Reply:
Sure, will update.